pkg/rollupsmachine/server.go

```diff
@@ -0,0 +1,126 @@
+// (c) Cartesi and individual authors (see AUTHORS)
+// SPDX-License-Identifier: Apache-2.0 (see LICENSE)
+
+package rollupsmachine
+
+import (
+	"fmt"
+	"io"
+	"log/slog"
+	"os/exec"
+	"regexp"
+	"strconv"
+
+	"github.com/cartesi/rollups-node/internal/linewriter"
+	"github.com/cartesi/rollups-node/pkg/emulator"
+)
+
+type ServerVerbosity string
+
+const (
+	ServerVerbosityTrace ServerVerbosity = "trace"
+	ServerVerbosityDebug ServerVerbosity = "debug"
+	ServerVerbosityInfo  ServerVerbosity = "info"
+	ServerVerbosityWarn  ServerVerbosity = "warn"
+	ServerVerbosityError ServerVerbosity = "error"
+	ServerVerbosityFatal ServerVerbosity = "fatal"
+)
+
+// StartServer starts a JSON RPC remote cartesi machine server.
+//
+// It configures the server's logging verbosity and initializes its address to localhost:port.
+// If verbosity is an invalid LogLevel, a default value will be used instead.
+// If port is 0, a random valid port will be used instead.
+//
+// StartServer also redirects the server's stdout and stderr to the provided io.Writers.
+//
+// It returns the server's address.
+func StartServer(verbosity ServerVerbosity, port uint32, stdout, stderr io.Writer) (string, error) {
+	// Configures the command's arguments.
+	args := []string{}
+	if verbosity.valid() {
+		args = append(args, "--log-level="+string(verbosity))
+	}
+	if port != 0 {
+		args = append(args, fmt.Sprintf("--server-address=localhost:%d", port))
+	}
+
+	// Creates the command.
+	cmd := exec.Command("jsonrpc-remote-cartesi-machine", args...)
```

Comment:
Is it possible to configure logs produced by the `json-rpc-server`, so they are the same format as Node ones?

Reply:
I don't know what you mean by embedding. But I do think we can prefix lines with the proper headers.
Adding this to the technical debt list.